src/workflow-design-guide/general-principles.rst

```diff
@@ -274,29 +256,34 @@ development and testing.
 Clock-Triggered Tasks
 ---------------------
 
-Tasks that wait on real time data should use clock-triggers to delay job
-submission until the expected data arrival time:
+Tasks that wait on real time data should use
+:ref:`clock triggers <Built-in Clock Triggers>`
+to delay job submission until the expected data arrival time:
 
 .. code-block:: cylc
 
    [scheduling]
        initial cycle point = now
        [[special tasks]]
            # Trigger 5 min after wallclock time is equal to cycle point.
-           clock-trigger = get-data(PT5M)
+           clock = wall_clock(offset=PT5M)
        [[graph]]
-           T00 = get-data => process-data
+           T00 = @clock => get-data => process-data
+
+.. cylc-scope:: flow.cylc[runtime][<namespace>]
 
 Clock-triggered tasks typically have to handle late data arrival. Task
-execution *retry delays* can be used to simply retrigger the task at
-intervals until the data is found, but frequently retrying small tasks probably
-should not go to a :term:`job runner`, and multiple task failures will be logged
-for what is a essentially a normal condition (at least it is normal until the
-data is really late).
+:cylc:conf:`execution retry delays` can be used to simply retrigger
+the task at intervals until the data is found, but frequently retrying small
+tasks should probably not go to a :term:`job runner`, and multiple task
```

Comment:
All tasks have to submit to a job runner, even if just local background one. I think the intention here was "probably should not go to a proper batch system / resource manager".

Reply:
Removed reference to job runner.